client/jetpack-cloud/sections/partner-portal/searchable-dropdown/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComboboxControl, Disabled } from '@wordpress/components';
+import classNames from 'classnames';
+
+import './style.scss';
+
+type Props = ComboboxControl.Props & {
+	disabled?: boolean;
+};
+
+export default function SearchableDropdown( props: Props ) {
+	const { disabled } = props;
+
+	return (
+		<div
+			className={ classNames( 'searchable-dropdown', {
+				'is-disabled': disabled,
+			} ) }
+		>
+			{ ! disabled && <ComboboxControl { ...props } /> }
+
+			{ disabled && (
+				<Disabled>
```

Comment:
If `Disabled` supports a `isDisabled` prop that you can use by passing `disabled` to it. That way you won't have to render two separate `ComboboxControl` instances.

Reply:
Well, the type definition includes `isDisabled` in trunk: https://github.com/WordPress/gutenberg/blob/b439e5f1a305f1c49d0817772aa0e7b02fa6c863/packages/components/src/disabled/types.ts#L7

It was added 9 months ago in #42708. 

The `@wordpress/components` package now ships its own types, so when we upgrade to the new version of `@wordpress/components`, it will be able to benefit from those types. We're currently working on the upgrade in #77121 and we're hoping to conclude it in the following weeks.

In the meantime, feel free to add a temporary type patch or even a `@ts-ignore` and a `TODO` comment there, pointing out that upgrading the `@wordpress/components` package to use its own types will fix it.

Reply:
Thanks for looking into this. Good to hear this will be fixed soon. I added the `ts-ignore` as suggested. 👍